odoo/orm/domains.py

```diff
@@ -540,22 +504,21 @@ class DomainNary(Domain):
     OPERATOR_SQL: SQL = SQL(" ??? ")
     ZERO: DomainBool = _FALSE_DOMAIN  # default for lint checks
 
-    __slots__ = ('children', '_opt_level')  # noqa: RUF023
+    __slots__ = ('children')  # noqa: RUF023
```

Comment:
```suggestion
    __slots__ = ('children',)
```
Not sure for the tuple, but IMO, it is preferable to be consistent with the `DomainBool`